app/models/infra_conversion_job.rb

```diff
@@ -263,7 +270,36 @@ def apply_right_sizing_memory(mode)
   def start
     migration_task.update!(:state => 'migrate')
     migration_task.update_options(:migration_phase => 'pre')
+    return queue_signal(:start_precopying_disks) if migration_task.warm_migration?
+
+    queue_signal(:wait_for_ip_address)
```

Comment:
Minor, but I think this would make more sense as
```
migration_task.warm_migration? ? queue_signal(:start_precopying_disks) : queue_signal(:wait_for_ip_address)
```